UefiPayloadPkg/Library/FdtParserLib/FdtParserLib.c

```diff
@@ -564,19 +567,22 @@ ParsegraphicNode (
 
   @param[in]  Fdt               Address of the Fdt data.
   @param[in]  SubNode           first Sub node of the PCI root bridge node.
+  @param[in]  AddressCells      #address-cells for serial port node 'reg' property.
+  @param[in]  SizeCells         #size-cells for serial port node 'reg' property.
 **/
 VOID
 ParseSerialPort (
-  IN VOID   *Fdt,
-  IN INT32  SubNode
+  IN VOID     *Fdt,
+  IN INT32    SubNode,
+  IN UINT32   AddressCells,
+  IN UINT32   SizeCells
   )

```

Comment:
It seems input SizeCells wasn't used in this function.
Can we remove it?

Reply:
Thanks for the comment. It was removed in the updated PR.